packages/manager/modules/pci/src/components/users/download-rclone/download-rclone.controller.js

```diff
@@ -24,21 +23,27 @@ export default class PciUsersDownloadRcloneController {
   }
 
   $onInit() {
-    this.isLoading = false;
+    this.isLoading = true;
     this.ovhFeatureFlipping
```

Comment:
Not in your changes, but this call should be preceeded by a `return` statement.
Also, the feature availability check could be done in a routing (this implies some refactor)

Reply:
Sure Jeremy. Can I do refractor with another PR and let this PR with this changes itself  ?

Reply:
Yes ofc